src/trio/_core/_tests/test_parking_lot.py

```diff
@@ -215,3 +218,73 @@ async def test_parking_lot_repark_with_count() -> None:
             "wake 2",
         ]
         lot1.unpark_all()
+
+
+async def test_parking_lot_breaker_basic() -> None:
+    lot = ParkingLot()
+    task = trio.lowlevel.current_task()
+
+    with pytest.raises(
+        RuntimeError,
+        match="Attempted to remove task as breaker for a lot it is not registered for",
+    ):
+        trio.lowlevel.remove_parking_lot_breaker(task, lot)
+    trio.lowlevel.add_parking_lot_breaker(task, lot)
+    trio.lowlevel.add_parking_lot_breaker(task, lot)
+    trio.lowlevel.remove_parking_lot_breaker(task, lot)
+    trio.lowlevel.remove_parking_lot_breaker(task, lot)
```

Comment:
add a code comment to explain these duplicated lines?

Reply:
```suggestion
    # check that breaking a parking lot is re-entrant
    trio.lowlevel.add_parking_lot_breaker(task, lot)
    trio.lowlevel.add_parking_lot_breaker(task, lot)
    trio.lowlevel.remove_parking_lot_breaker(task, lot)
    trio.lowlevel.remove_parking_lot_breaker(task, lot)
```
(that's actually a bad explanation maybe, but it's not like test comments matter that much)

Reply:
added comment, slightly different from a5rocks suggestion